recipes-wpe/wpewebkit/wpewebkit.inc

```diff
@@ -76,7 +76,7 @@ PACKAGECONFIG[woff2] = "-DUSE_WOFF2=ON,-DUSE_WOFF2=OFF,woff2"
 
 
 # DRM
-PACKAGECONFIG[opencdm] = "-DENABLE_OPENCDM=ON,-DENABLE_OPENCDM=OFF,wpeframework"
+PACKAGECONFIG[opencdm] = "-DENABLE_OPENCDM=ON,-DENABLE_OPENCDM=OFF,wpeframework-clientlibraries"
```

Comment:
I think we have to give wpeframework-clientlibraries dependency with wpebackend-rdk in case of backend as wpeframework

Reply:
Done.